handler_worlds.go

```diff
@@ -259,11 +259,13 @@ func WorldZip(owner, name string, user *users.User, tx *gorm.DB,
 
 	// commit the DB transaction
 	// Note: we commit the TX here on purpose, to be able to detect DB errors
-	// before writing "data" to ResponseWriter. Once you write data (not headers)
+	// before writing "data" to ResponseWriter.
 	if err := tx.Commit().Error; err != nil {
 		return nil, gz.NewErrorMessageWithBase(gz.ErrorZipNotAvailable, err)
 	}
 
+	// If ?link=true, fuel will return a link to a cloud storage where the client can perform a subsequent request
+	// to download the resource. If ?link=false or if it is not included, it will serve the file directly to the client.
```

Comment:
Abstract away the details of how `linkRequested` is defined. You can move the specifics (i.e. `if ?link=true`) to the `isLinkRequeted` documentation.

```suggestion
	// If a link was requested, fuel will return a link to a cloud storage where the client can perform a subsequent request
	// to download the resource. If a link was not requested or if it is not included, it will serve the file directly to the client.
```